main/wallet.js

```diff
@@ -30,6 +30,7 @@ backend.transactions = loadStoredEntries()
 /** @type {Context | null} */
 let ctx = null
 let balance = loadBalance()
+const scheduledRewards = loadScheduledRewards()
```

Comment:
since this is a const, doesn't that mean `getScheduledRewards` will always return the same number, independent of `updateScheduledRewards`?

Reply:
Great catch! I used `let` but VS Code auto-fixed this to `const`. I need to double-check why we are not updating this variable.

Reply:
Fixed in cae4a42